lib/commands/misc/rmrole.js

```diff
@@ -0,0 +1,53 @@
+const { Command } = require('discord.js-commando');
+const { GENERAL_ID } = require('../../constants.js');
+const { getChannel } = require('./../../util.js');
+
+module.exports = class RemoveRoleCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'rmrole',
+			group: 'misc',
+			memberName: 'rmrole',
+			description: 'Remove a self-assigned role from yourself.',
+			details: '',
+			args: [
+				{
+					key: 'role',
+					prompt: 'Remove a role!',
+					type: 'string',
+					infinite: false,
+					wait: 1
+				}
+			]
+		});
+		this.queues = {};
+	};
+
+  async run(msg, args) {
+    const channel = this.client.channels.get(getChannel(REOL_CHAN_ID));
+    const role = msg.guild.roles.find(r => r.name == args.role);
+    const memberRole = msg.guild.roles.find(r => r.name == 'Member');
+
+    // Prevent removal of required roles (Can be updated in the future if more required roles are ever added).
+    if (args.role === "Member" || args.role === "member") {
```

Comment:
the role is called `Member`, so I don't think we need the second option. Since you already have `const role` above, you could use that instead of `args.role`.. maybe something like `role === memberRole`?

Reply:
either way, it's probably also best lower case the role and the string literal we're comparing to. e.g. `args.role.toLowerCase() === 'member'`